signer/src/transaction_coordinator.rs

```diff
@@ -540,6 +541,13 @@ where
             )
             .await?;
 
+            metrics::counter!(
```

Comment:
nit: should we also add a counter for the number of deposits swept?

Reply:
Done in https://github.com/stacks-network/sbtc/pull/1084/commits/10b563c5e15aa361b2a56e2857c7d69e5d4e7e42